pkg/scheduler/framework/plugins/helper/node_affinity.go

```diff
@@ -56,16 +56,16 @@ func NodeMatchesNodeAffinity(affinity *v1.NodeAffinity, node *v1.Node) bool {
 	// }
 
 	// Match node selector for requiredDuringSchedulingIgnoredDuringExecution.
-	if affinity.RequiredDuringSchedulingIgnoredDuringExecution != nil && !nodeMatchesNodeSelector(node, affinity.RequiredDuringSchedulingIgnoredDuringExecution) {
+	if affinity.RequiredDuringSchedulingIgnoredDuringExecution != nil && !NodeMatchesNodeSelector(node, affinity.RequiredDuringSchedulingIgnoredDuringExecution) {
 		return false
 	}
 	return true
 }
 
 // nodeMatchesNodeSelector checks if a node's labels satisfy a list of node selector terms,
 // terms are ORed, and an empty list of terms will match nothing.
-func nodeMatchesNodeSelector(node *v1.Node, nodeSelector *v1.NodeSelector) bool {
+func NodeMatchesNodeSelector(node *v1.Node, nodeSelector *v1.NodeSelector) bool {
```

Comment:
rather, we should be able to get rid of this function now. And perhaps the entire file.